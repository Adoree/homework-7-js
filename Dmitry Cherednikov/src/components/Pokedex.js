import React from 'react';

import Header from './Header';
import MainContent from './MainContent';

const Pokedex = () => (
  <div>
    <Header />
    <MainContent />
  </div>
);

export default Pokedex;