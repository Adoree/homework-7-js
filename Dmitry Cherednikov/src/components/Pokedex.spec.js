import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Pokedex from './Pokedex';

describe('<Pokedex />', () => {
  it('should render correctly', () => {
    const output = shallow((
      <Pokedex />
    ));

    expect(toJson(output)).toMatchSnapshot();
  })
});