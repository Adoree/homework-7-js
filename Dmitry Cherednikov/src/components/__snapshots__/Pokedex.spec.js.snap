// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pokedex /> should render correctly 1`] = `
<div>
  <Header />
  <MainContent />
</div>
`;
